cmd/beacon-chain/BUILD.bazel

```diff
@@ -134,12 +67,12 @@ go_test(
 )
 
 prysm_image_upload(
-    name = "push_oci_image",
+    name = "push_images",
     binary = ":beacon-chain",
     entrypoint = ["/beacon-chain"],
-    repository = "gcr.io/prylabs-dev/prysm/beacon-chain",
+    repository = "gcr.io/prysmaticlabs/prysm/beacon-chain",
```

Comment:
this isn't related right? this just fixes the namespace?

Reply:
This part is the "make multiarch images canonical" half of the PR.